subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/javadoc/JavadocCachingIntegrationTest.groovy

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.gradle.javadoc
+package org.gradle.api.tasks.javadoc
```

Comment:
These `Javadoc` integration tests were living in a package that was not corresponding to the package of the `Javadoc` task itself as it is done for other task integration tests